test/update-user.test.js

```diff
@@ -37,61 +38,63 @@ suite('seneca-user update suite tests ', function () {
     })
   })
 
-  test('user/register test', function (done) {
+  test('user/register test user1', function (done) {
     si.act(_.extend({role: 'user', cmd: 'register'}, user1Data), function (err, data) {
-      assert.isNull(err)
-      assert(user1Data.nick, data.nick)
+      expect(err).to.not.exist()
+      expect(data.nick).to.not.exist()
+      // FIXME: check if good test
```

Comment:
Seems like we would want

```js
expect(data.nick).to.equal(user1Data.nick)
```

right?

To test that the user we get back is the same one we registered